iot-hub/Samples/service/FileUploadNotificationReceiverSample/FileUploadNotificationReceiverSample.cs

```diff
@@ -79,7 +79,7 @@ private async Task ReceiveFileUploadNotifications(string targetDeviceId, Cancell
             {
                 try
                 {
-                    FileNotification fileUploadNotification = await notificationReceiver.ReceiveAsync(s_notificationReceiverTimeout);
```

Comment:
Is s_notificationReceiverTimeout being used anywhere else? If not, we could remove it.

Reply:
Apart from the declaration, it seems like the only other place that it's used is on line 86 where the value is logged. I'm about the best way to proceed, and if it's still needed.